src/vs/workbench/contrib/notebook/browser/notebook.css

```diff
@@ -17,6 +17,10 @@
 	white-space: initial;
 }
 
+.scrollable {
```

Comment:
This should be scoped down - the name is too generic and likely to be used elsewhere in the application.

Reply:
Yeah, sorry about that. I just fixed this by scopy it to notebook-editor in the sql path instead.